cli/pcluster/configure/utils.py

```diff
@@ -28,8 +28,14 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
         except (BotoCoreError, ClientError) as e:
             LOGGER.error("Failed with error: %s" % e)
-            LOGGER.error("Hint: please check your AWS credentials.")
-            LOGGER.error("Run `aws configure` or set the credentials as environment variables.")
+            if isinstance(e, ClientError) and "credentials" in str(e):
+                LOGGER.error("Hint: please check your AWS credentials.")
```

Comment:
@nstornetta had some suggestions here

Reply:
My message is below this comment. I kept the error message for credentials exception and wrote a new message for NoRegionError exception The message was written according to @nstornetta's advice

Reply:
We can improve it to be similar to the new one. 
Something like:
`Missing AWS Credentials, run 'aws configure' or set them as environment variables`